blog about monitoring deprecation (#925)
kyma-project/website: content/blog-posts/2022-11-02-loki-deprecation/index.md

## The Background

> Kyma shifted the focus towards providing enterprise-grade components as building blocks that you can use to connect your productive workloads with the SAP ecosystem. Kyma's classic in-cluster Logging component does not qualify for enterprise-grade criteria because it does not scale with your application log load. Furthermore, you cannot configure it in any way, like for data retention times. Lastly, running a logging backend in-cluster (in the same cluster where your workload is running) has several flaws:

Reviewer: ```suggestion
Kyma shifted the focus towards providing enterprise-grade components as building blocks that you can use to connect your production workloads with the SAP ecosystem. Kyma's classic in-cluster Logging component does not qualify for enterprise-grade criteria because it does not scale with your application log load. Furthermore, you cannot configure it in any way, like for data retention times. Lastly, running an in-cluster logging backend (in the same cluster where your workload is running) has several drawbacks:
```
Let's keep it aligned with the Monitoring deprecation
